src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -1104,11 +1104,10 @@ private static string GetFileName_Windows(string path)
         /// </summary>
         private static string GetFileName_Unix(string path)
         {
-            int length = path.Length;
-            for (int i = length; --i >= 0;)
-                if (path[i] == '/')
-                    return path.Substring(i + 1);
-            return path;
+            int i = path.LastIndexOf('/');
```

Comment:
Would it make sense to also change `GetFileName_Windows` above to use `LastIndexOfAny`?

Reply:
Yup.  Hadn't noticed that one.